api/proto/teleport/legacy/types/types.proto

```diff
@@ -5197,6 +5199,16 @@ message PluginOpsgenieAccessSettings {
   string api_endpoint = 5;
 }
 
+message PluginPagerDutyAccessSettings {
+  option (gogoproto.equal) = true;
+
+  // PagerDutyUserEmail is the email address of the PagerDuty user that
+  // will be listed as the reporter source of incidents, comments, etc
+  // within PagerDuty. Should usually be the same user the API key
+  // represents.
+  string pager_duty_user_email = 1;
```

Comment:
```suggestion
  string user_email = 1;
```

Reply:
Yeah, I had originally used `user_email` - but I wanted to make it crystal clear that this was the the email for the _PagerDuty_ user, as opposed to the Teleport user,